x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_list.tsx

```diff
@@ -165,42 +150,29 @@ export const PolicyList = React.memo(() => {
   );
 
   return (
-    <EuiPage data-test-subj="policyListPage">
-      <EuiPageBody>
-        <EuiPageContent>
-          <EuiPageContentHeader>
-            <EuiPageContentHeaderSection>
-              <EuiTitle size="l">
-                <h1 data-test-subj="policyViewTitle">
-                  <FormattedMessage
-                    id="xpack.endpoint.policyList.viewTitle"
-                    defaultMessage="Policies"
-                  />
-                </h1>
-              </EuiTitle>
-              <h2>
-                <EuiText color="subdued" data-test-subj="policyTotalCount" size="s">
-                  <FormattedMessage
-                    id="xpack.endpoint.policyList.viewTitleTotalCount"
-                    defaultMessage="{totalItemCount} Policies"
-                    values={{ totalItemCount }}
-                  />
-                </EuiText>
-              </h2>
-            </EuiPageContentHeaderSection>
-          </EuiPageContentHeader>
-          <EuiPageContentBody>
-            <EuiBasicTable
-              items={policyItems}
-              columns={columns}
-              loading={loading}
-              pagination={paginationSetup}
-              onChange={handleTableChange}
-              data-test-subj="policyTable"
-            />
-          </EuiPageContentBody>
-        </EuiPageContent>
-      </EuiPageBody>
-    </EuiPage>
+    <PageView
+      data-test-subj="policyListPage"
+      headerLeft={i18n.translate('xpack.endpoint.policyList.viewTitle', {
+        defaultMessage: 'Policies',
+      })}
+      bodyHeader={
+        <EuiText color="subdued" data-test-subj="policyTotalCount">
```

Comment:
I think you need to adjust tests for checking for another `data-test-subj`, it checks for `policyViewTitle` now.  We need to readjust

Reply:
Yes, agreed. They are actually currently failing, but I wanted to hold off on fixing them until I got feedback that these changes look good to go in. :)

Reply:
the PageView component certainly lgtm, might have questions about the router and hook changes still